### PR TITLE
drupal: add page

### PR DESCRIPTION
**Pull request by prudloff-insite**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/drupal.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- Uninstall module "foo":
+
+`drupal module:uninstall {{foo}}`
+
+- Clear all caches:
+
+`drupal cache:rebuild`
+
+- View current Drupal Installation status:
```
Comment (einverne) [inline pages/common/drupal.md]: Should `**I**nstallation` be **i**nstallation here?

File: pages/common/drupal.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# drupal
+
+> Drupal Console.
+> The Drupal CLI. A tool to generate boilerplate code, interact with and debug Drupal.
+> More information: <https://drupalconsole.com/>.
+
+- Install module "foo":
```
Comment (agnivade) [inline pages/common/drupal.md]: Perhaps it's better to say "Install module:" and in the example, change it to "module_name" from "foo". And do the same for next one too.

File: pages/common/drupal.md (hunk 1 above)
Comment (agnivade) [reply pages/common/drupal.md]: Yes, I think so.

File: pages/common/drupal.md (hunk 2 above)
Comment (prudloff-insite) [reply pages/common/drupal.md]: I tried to use the same syntax as the `drush` page. But I agree `{{module_name}}` is easier to understand.

File: pages/common/drupal.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# drupal
+
+> Drupal Console.
```
Comment (sbrl) [inline pages/common/drupal.md]: Perhaps this will read better?
```suggestion
> CLI for Drupal.
```

File: pages/common/drupal.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+> The Drupal CLI. A tool to generate boilerplate code, interact with and debug Drupal.
```
Comment (sbrl) [inline pages/common/drupal.md]: Perhaps this will read better?
```suggestion
> Generates boilerplate code, interacts with and debugs Drupal projects.
```

**Comment by sbrl** [comment]
Thanks, @prudloff-insite!

Merging, since @einverne's comment has been addressed.

**Pull request merged**